x-pack/plugin/core/src/test/java/org/elasticsearch/test/SecuritySettingsSourceField.java

```diff
@@ -22,9 +25,24 @@ public final class SecuritySettingsSourceField {
               allow_restricted_indices: true
               privileges: [ "ALL" ]
           run_as: [ "*" ]
-          # The _es_test_root role doesn't have any application privileges because that would require loading data (Application Privileges)
-          # from the security index, which can causes problems if the index is not available
+          applications:
+            - application: "*"
+              privileges: [ "*" ]
+              resources: [ "*" ]
```

Comment:
This is no longer the case. Added these application privilege so this role definition is consistent with the one used for testclusters.